backend/src/common/enums/instance-type/index.ts

```diff
@@ -0,0 +1 @@
+export * from 'healthcare-shared/common/enums/intance-type';
```

Comment:
From the `shared` folder you should export specific things
`export { something  } from 'healthcare-shared/common/enums/intance-type'`